### PR TITLE
Mainnet deployment documentation

### PR DESCRIPTION
**Pull request by albertandrejev**

### CODE REVIEW

File: docs/deployments/overview.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+   ```
+
+4. Compare the obtained checksums to confirm integrity.
+
+## Contracts and Their Checksums
+
+The contracts were built using the git commit hash `b1f986c9b7a45ab9dc21ee7c86e48bbf31fdc928`.
+You can find contracts checksums, code id and addresses on the next two pages: [testnet](./testnet.md) and [mainnet](./mainnet.md).
+
+## Conclusion
```
Comment (oldremez) [inline docs/deployments/overview.md:56]: I think conclusion is redundant here

File: docs/deployments/overview.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Overview
+
+## Introduction
+
+To ensure the integrity and security of contracts uploaded to the network, it is necessary to verify the SHA256 checksums of both the local binary file of the contract and the contract deployed on the network. This document outlines the steps for performing this verification.
+
+## Tools
```
Comment (oldremez) [inline docs/deployments/overview.md:7]: Please add
1. how to find proper version of the contract
2. how to build contracts

File: docs/deployments/overview.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+1. **Command to obtain the checksum of the file:**
+
+   ```
+   sha256sum <contract file name>
+   ```
+
+2. **Command to obtain the code id of the deployed contract:**
+
+   ```
+   neutrond q wasm contract <address of the deployed contract> --node https://rpc-talzor.neutron-1.neutron.org:443
```
Comment (oldremez) [inline docs/deployments/overview.md:18]: Please add Celatone as an alternative

**Review by oldremez — CHANGES_REQUESTED** [review]

File: docs/deployments/overview.md (hunk 2 above)
Comment (albertandrejev) [reply docs/deployments/overview.md:7]: what do you mean by contract version? Git commit that was used to build binaries? Information about it in the "Contracts and Their Checksums" section, but I can reorganize information

**Pull request merged**